openapi/openapi.yaml

```diff
@@ -257,6 +305,98 @@ paths:
 
 components:
   schemas:
+
+    Collection:
+      title: 'Collection'
+      description: 'Ansible content collection'
+      type: object
+      properties:
+        name:
+          description: 'Collection name. Must be lower case containing
+                        only alphanumeric characters and underscores.'
+          type: string
+          maxLength: 64
+          readOnly: true
+        remote_id:
+          description: 'Collection ID in remote database'
+          type: string
+          format: uuid
+          readOnly: true
+        quality_score:
```

Comment:
quality_score is not supported for Automation Hub API.

Reply:
Agreed, didn't seem like it makes sense here. That was based on https://github.com/ansible/galaxy-api/pull/4/files#diff-501a88204e6d0fc39ef5ad9debd9e98dR86 so may need updating there (or is it just not going to be exposed via the API?)

Reply:
updated